packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -679,31 +685,55 @@ class _PlatformViewLinkState extends State<PlatformViewLink> {
   PlatformViewController _controller;
   bool _platformViewCreated = false;
   Widget _surface;
+  FocusNode _focusNode;
 
   @override
   Widget build(BuildContext context) {
     if (!_platformViewCreated) {
       return const SizedBox.expand();
     }
     _surface ??= widget._surfaceFactory(context, _controller);
-    return _surface;
+    return Focus(
+      focusNode: _focusNode,
+      onFocusChange: _handleFrameworkFocusChanged,
+      child: _surface,
+    );
   }
 
   @override
   void initState() {
+    _focusNode = FocusNode(debugLabel: 'PlatformView(id: $_id)',);
     _initialize();
     super.initState();
   }
 
   void _initialize() {
     _id = platformViewsRegistry.getNextPlatformViewId();
-    _controller = widget._onCreatePlatformView(PlatformViewCreationParams._(id:_id, onPlatformViewCreated:_onPlatformViewCreated));
+    _controller = widget._onCreatePlatformView(
+      PlatformViewCreationParams._(
+        id:_id,
+        onPlatformViewCreated:_onPlatformViewCreated,
+        onFocusChanged:_handlePlatformFocusChanged
+      ),
+    );
   }
 
   void _onPlatformViewCreated(int id) {
     setState(() => _platformViewCreated = true);
   }
 
+  void _handleFrameworkFocusChanged(bool isFocused) {
+    if (!isFocused) {
+      _controller?.clearFocus();
```

Comment:
Let's set `_controller` to null after calling dispose, to make sure we're not accidentally doing this on a disposed controller.

Reply:
done! Going to merge on green. Thanks for the quick turnaround.